resources/guis.yml

```diff
@@ -256,12 +256,12 @@ guis:
   price: Free
   license: MIT
   order: 28
-- name: GitAtomic
-  url: https://gitatomic.bitbucket.io/
-  image_tag: guis/gitatomic@2x.png
+- name: Cong
```

Comment:
Just for curiosity, why/how did you change the order of these clients? :)

Reply:
I followed the instructions I followed the 'sort the tools' instructions in the readme here https://github.com/git/git-scm.com#adding-new-gui